docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -864,6 +865,28 @@ https://godbolt.org/z/6qTPz9n6h
 
 ---
 
+### Vtable pointers check
```

Comment:
Use the first sentence of short description from Table 2 as title here to be consistent with the other option descriptions.